src/scripts/datepicker.js

```diff
@@ -70,9 +73,7 @@ export class DatePicker {
     show() {
         let parentOffset = this.parentElement.getBoundingClientRect();
         this.isOpened = true;
-        this.datepicker.style.display = 'block';
-        this.datepicker.style.top = `${parentOffset.top + 55}px`;
-        this.datepicker.style.left = `${parentOffset.left}px`;
+        this.datepicker.style.cssText = `display: block; top: ${(parentOffset.top + 55)}px; left: ${parentOffset.left}px;`;
```

Comment:
@diasbruno @evangilo  Ok, but you must minimise the repaint and reflow and this line just have three styles declarations.